src/Components/Notification.vue

```diff
@@ -267,7 +267,9 @@ export default {
 			emit('notifications:notification:received', event)
 		}
 
-		if (this.shouldNotify && this.$parent.$parent.$parent.showBrowserNotifications) {
+		if (this.shouldNotify
+			&& this.$parent.$parent.$parent.showBrowserNotifications
+			&& this.$parent.$parent.$parent.webNotificationsThresholdId < this.notificationId) {
```

Comment:
> you changing webNotificationsThresholdId before and right after the _fetch, so Vue components for new notifications might not be mounted yet.

Before was added "later" and should not be an issue.

After is no problem as the first run (`!this.backgroundFetching` which is wrapping it) is not generating notifications anyway. I was just afraid to remove this, in case someone would manually delete/resolve all of the initial load. Then the first background fetch would have an empty notification list and therefore there would still be sound+notifications for old things that are loaded with the first background fetch